test_runner/src/main/kotlin/ftl/args/ArgsHelper.kt

```diff
@@ -265,6 +265,23 @@ object ArgsHelper {
 
         return shards.map { Chunk(find + it.testMethods.filterNot { method -> find.contains(method) }) }
     }
+
+    fun calculateDummyShards(
+        filteredTests: List<FlankTestMethod>,
+        args: AndroidArgs,
+    ): CalculateShardsResult {
+        if (filteredTests.isEmpty()) {
+            return CalculateShardsResult(emptyList(), emptyList()) // Avoid unnecessary computing if we already know there aren't tests to run.
+        }
+        val (ignoredTests, _) = filteredTests.partition { it.ignored }
```

Comment:
I think we could skip this part. As result, we will have duplicated 'ignored' tests in XML results